dji_interface/include/dji_interface/dji_interface.h

```diff
@@ -35,11 +35,13 @@
 #include <mav_msgs/Status.h>
 #include <sensor_msgs/Imu.h>
 #include <sensor_msgs/Joy.h>
+#include <sensor_msgs/NavSatFix.h>
 
 #include <Eigen/Eigen>
 
 #include <dji_comm/dji_comm.h>
-
+#define C_PI (double)3.141592653589793
```

Comment:
please use `constexpr` instead of `define`

Reply:
LGTM.